ubuntu-advantage.1

```diff
@@ -114,6 +114,24 @@ The logging level used when writing to \fBlog_file\fP
 .B
 \fBlog_file\fP
 The log file for the Ubuntu Advantage client
+.TP
+.B
+\fBhttp_proxy\fP
```

Comment:
In docs, we might have to define a `ua_config` section with these keys under it maybe? not quite sure, but leave the decision to you.